src/app/static/src/app/components/ChoroplethFilters.vue

```diff
@@ -0,0 +1,160 @@
+<template>
```

Comment:
I haven't tried to reuse code between this and the once-and-future Filters component, as it felt like they would diverge enough to not be worth it.